src/hotspot/share/memory/universe.cpp

```diff
@@ -133,11 +154,11 @@ OopHandle Universe::_virtual_machine_error_instance;
 OopHandle Universe::_reference_pending_list;
 
 Array<Klass*>* Universe::_the_array_interfaces_array = nullptr;
-LatestMethodCache* Universe::_finalizer_register_cache = nullptr;
-LatestMethodCache* Universe::_loader_addClass_cache    = nullptr;
-LatestMethodCache* Universe::_throw_illegal_access_error_cache = nullptr;
-LatestMethodCache* Universe::_throw_no_such_method_error_cache = nullptr;
-LatestMethodCache* Universe::_do_stack_walk_cache     = nullptr;
+LatestMethodCache Universe::_finalizer_register_cache;
+LatestMethodCache Universe::_loader_addClass_cache;
+LatestMethodCache Universe::_throw_illegal_access_error_cache;
+LatestMethodCache Universe::_throw_no_such_method_error_cache;
+LatestMethodCache Universe::_do_stack_walk_cache;
```

Comment:
Just make these LatestMethodCache and not static data members in Universe, and move them up with the LatestMethodCache type declaration so there're all in one place.

Reply:
Done. Thanks for the suggestion.